openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -541,6 +546,30 @@ const GlossaryV1 = ({
               </Space>
             )}
           </div>
+          <Space className="m-b-md" size={8}>
+            <Typography.Text className="text-grey-muted">
+              {t('label.updated-by')} -
+            </Typography.Text>
+            <Space size={8}>
+              <ProfilePicture
+                displayName={selectedData.updatedBy}
+                id={selectedData.id}
+                name={selectedData.updatedBy || ''}
+                textClass="text-xs"
+                width="20"
+              />
+              <span>
+                <Link to={getUserPath(selectedData.updatedBy ?? '')}>
+                  {selectedData.updatedBy}
+                </Link>
+                {` ${t('label.on-lowercase')} 
+               ${getDateTimeByTimeStamp(
+                 selectedData.updatedAt || 0,
+                 'dd MMM yyyy, hh:mm a'
+               )}`}
```

Comment:
We have `formatDateTime` utils, so use that here.

Reply:
I am already using util method! :)